lib/ASRouterTargeting.jsm

```diff
@@ -422,6 +428,9 @@ const TargetingGetters = {
       false
     );
   },
+  get totalBlockedCount() {
```

Comment:
For a very new profile this could be `0` so the message doesn't make much sense. The date is also messed up and returns the `0` timestamp for `Since ...` subtitle.